opentracing-mock/src/test/java/io/opentracing/mock/MockTracerTest.java

```diff
@@ -64,21 +62,15 @@ public void testRootSpan() {
         assertEquals(7, tags.get("int"));
         assertEquals("foo", tags.get("string"));
         List<MockSpan.LogEntry> logs = finishedSpan.logEntries();
-        assertEquals(3, logs.size());
+        assertEquals(2, logs.size());
         {
             MockSpan.LogEntry log = logs.get(0);
-            assertEquals(1001, log.timestampMicros());
-            assertEquals("event name", log.fields().get("event"));
```

Comment:
unless there is another test elsewhere, I would keep this test but instead of log(event, payload) only test log(event)